website/docs/r/cognito_user_pool.markdown

```diff
@@ -31,7 +31,11 @@ The following arguments are supported:
 * `email_configuration` (Optional) - The [Email Configuration](#email-configuration).
 * `name` - (Required) The name of the user pool.
 * `email_verification_subject` - (Optional) A string representing the email verification subject.
+
+~> **Note** `verification_message_template.email_subject` replaces this string when you specify these two.
```

Comment:
I'm wondering whether we should set email_verification_subject as `ConflictsWith: []string{"verification_message_template.0. email_subject"},`, since we cannot have both being different

Thus, we could avoid such notice... what do you think @atsushi-ishibashi 

Also pinging @radeksimko to get it's opinion :)

Reply:
I'm worrying that it becomes breaking-change...
There are users who are forced update `.tf` files.

Reply:
Yes, it would be, but since it's one or the other according to the AWS support... this is a required thing IMO.

Reply:
I got it👍

Reply:
@atsushi-ishibashi Correct me if I'm wrong but the API misbehaves and causes spurious diffs if you provide both?

The only case when I see `ConflictsWith` causing problems/BC is when folks use the exact same values.

We can defer the `ConflictsWith` addition to 2.0.0 and document this as suggested in the meantime.

@Ninir what do you think?

Reply:
> @atsushi-ishibashi Correct me if I'm wrong but the API misbehaves and causes spurious diffs if you provide both?

Exactly!

> We can defer the ConflictsWith addition to 2.0.0 and document this as suggested in the meantime.

Do agree on that :) 👍

Reply:
@radeksimko That's right!
> We can defer the ConflictsWith addition to 2.0.0 and document this as suggested in the meantime.

I also agree!